src/Entity/Packaging.php

```diff
@@ -5,6 +5,9 @@
 use Doctrine\DBAL\Types\Types;
 use Doctrine\ORM\Mapping as ORM;
 
+/**
+ * The box available in warehouse
+ */
```

Comment:
```suggestion
/**
 * Represents a set of boxes available in the warehouse.
 * <p />
 * Warehouse workers pack a set of products for a given order into one of these boxes.
 */
```

Reply:
> Represents a set of boxes available in the warehouse.

I feel we should describe what a row represents, not what the table represents.

Reply:
True, we can remove the "set of boxes" part. I still feel like we should explain it more in-depth if this is a reoccurring problem. 

As saying "it's a box in the warehouse" seems equivalent to putting "packaging" in the translator 😄